playbooks/common/openshift-cluster/upgrades/pre/verify_health_checks.yml

```diff
@@ -11,3 +11,6 @@
       checks:
       - disk_availability
       - memory_availability
+      - docker_image_availability
+      - etcd_imagedata_size
```

Comment:
checks can be disabled...

but I guess the question is really "are we running health checks generally or just looking for issues we think might impact the upgrade?"

to the extent that an upgrade might exercise data in etcd, the etcd checks probably make sense either way. you do not want your upgrade to fail partway through due to running out of etcd space, although it was probably going to fill up imminently anyway, at least you don't want to have to deal with that in the middle of an upgrade.